0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -98,47 +100,46 @@ Ledger protocols are responsible for executing the individual transfers that con
 
 #### Minimal Support
 
-For minimal Interledger support, the ledger MUST have the ability for funds to be transferred from one account to another. This enables OTP transactions to pass through this ledger. It also allows third parties to act as escrow providers in order to enable UTP and ATP transactions on the ledger.
+For minimal Interledger support, the ledger MUST have the ability for funds to be transferred from one account to another. All of the more advanced ledger layer functionality, such as escrowed transfers, messaging, etc. can be implemented on top of this type of basic ledger.
 
 #### Basic Support
 
 For basic Interledger support, a ledger MUST fulfill the requirements for minimal support and also the following:
 
-The ledger MUST provide cryptographic escrow. Escrow means that transfers can have four states:
+The ledger MUST provide cryptographic authorization holds. During an authorization hold, money is put aside for a specific transfer until that transfer's outcome has been decided.
+
+Transfers using authorization holds can be in four distinct states:
 
-![enter image description here](https://lh3.googleusercontent.com/-QHsehrPTXgM/Vu_zs1dLWPI/AAAAAAAAEys/zchHKJj_MQs/s0/Transfer%252520States.png "Transfer States")
+![Transfers start in the proposed state, transition to the prepared state and finally to the executed state, which is final. Proposed and prepared transfers can also transition to the rejected state, which is final as well.](../shared/graphs/transfer-states.svg)
 
 * Proposed -- Nothing has happened yet.
-* Prepared -- Funds are held in escrow.
+* Prepared -- Funds are held.
 * Executed -- The transfer has completed.
 * Rejected -- The transfer has been canceled (and funds returned to the sender.)
 
-<span class="show alert alert-info">**Hint:** Escrow is the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
+<span class="show alert alert-info">**Hint:** Authorization holds are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
 
-
-The ledger MUST be able to verify a simple SHA-256 hashlock cryptographic escrow condition in order to automatically trigger the release of the escrow. It must also allow a timeout to automatically roll back an escrowed transfer.
+The ledger MUST be able to release the held funds to the receiver upon receiving a 32-byte preimage whose SHA-256 hash matches a value provided by the sender. It MUST also support releasing the funds back to the sender after a timeout.
```

Comment:
For a ledger to be supported, it should accept ONLY 32-byte pre-images. Otherwise a connector could stand to lose money, if they have an outgoing transfer on a ledger with a larger fulfillment size than the ledger of the corresponding incoming transfer.